tests/parm/input_global_parent.nml.IN

```diff
@@ -0,0 +1,410 @@
+  consv_am = .false.
+  fill = .true.
+  full_zs_filter = @[FULL_ZS_FILTER]
+  dwind_2d = .false.
+  print_freq = 6
+  warm_start = @[WARM_START]
+  no_dycore = .false.
+  z_tracer = .true.
+  agrid_vel_rst = .true.
+  read_increment = @[READ_INCREMENT]
+  res_latlon_dynamics = @[RES_LATLON_DYNAMICS]
+  do_schmidt = .true.
+  target_lat = @[TARGET_LAT]
+  target_lon = @[TARGET_LON]
+  stretch_fac = 1.0001
+/
+
+&external_ic_nml
+  filtered_terrain = .true.
+  levp = 128
+  gfs_dwinds = .true.
+  checker_tr = .false.
+  nt_checker = 0
+  
+/
+
+&surf_map_nml
+  cd2 = -1
+  cd4 = 0.15
+  max_slope = 0.15
+  n_del2_strong = 0
+  n_del2_weak = 15
+  n_del4 = 2
+  peak_fac = 1.
+  zero_ocean = .false.
+/
+
+&gfs_physics_nml
+  fhzero       = @[FHZERO]
+  h2o_phys     = @[H2O_PHYS]
+  ldiag3d      = @[LDIAG3D]
+  qdiag3d      = @[QDIAG3D]
+  lightning_threat = @[LIGHTNING_THREAT]
+  print_diff_pgr = @[PRINT_DIFF_PGR]
+  fhcyc        = @[FHCYC]
+  use_ufo      = .true.
+  pre_rad      = .false.
+  imp_physics  = @[IMP_PHYSICS]
+  iovr         = @[IOVR]
+  lcnorm       = @[LCNORM]
+  ltaerosol    = .true.
+  lradar       = .true.
+  ttendlim     = -999
+  dt_inner     = @[DT_INNER]
+  sedi_semi    = @[SEDI_SEMI]
+  decfl        = @[DECFL]
+  oz_phys      = @[OZ_PHYS_NEW]
+  oz_phys_2015 = @[OZ_PHYS_OLD]
+  lsoil_lsm    = @[LSOIL_LSM]
+  do_mynnedmf  = @[DO_MYNNEDMF]
+  do_mynnsfclay = @[DO_MYNNSFCLAY]
+  icloud_bl    = 1
+  bl_mynn_edmf = 1
+  bl_mynn_tkeadvect = @[BL_MYNN_TKEADVECT]
+  bl_mynn_edmf_mom = 1
+  do_ugwp      = @[DO_UGWP]
+  do_tofd      = @[DO_TOFD]
+  do_ca        = @[DO_CA]
+  gwd_opt      = @[GWD_OPT]
+  do_ugwp_v0   = @[DO_UGWP_V0]
+  do_ugwp_v1   = @[DO_UGWP_V1]
+  do_ugwp_v0_orog_only = @[DO_UGWP_V0_OROG_ONLY]
+  do_ugwp_v0_nst_only  = @[DO_UGWP_V0_NST_ONLY]
+  do_gsl_drag_ls_bl    = @[DO_GSL_DRAG_LS_BL]
+  do_gsl_drag_ss       = @[DO_GSL_DRAG_SS]
+  do_gsl_drag_tofd     = @[DO_GSL_DRAG_TOFD]
+  do_ugwp_v1_orog_only = @[DO_UGWP_V1_OROG_ONLY]
+  do_ugwp_v1_w_gsldrag = @[DO_UGWP_V1_W_GSLDRAG]
+  min_lakeice  = @[MIN_LAKEICE]
+  min_seaice   = @[MIN_SEAICE]
+  use_cice_alb = @[USE_CICE_ALB]
+  pdfcld       = .false.
+  fhswr        = @[FHSWR]
+  fhlwr        = @[FHLWR]
+  ialb         = @[IALB]
+  iems         = @[IEMS]
+  iaer         = @[IAER]
+  icliq_sw     = @[ICLIQ_SW]
+  ico2         = 2
+  isubc_sw     = 2
+  isubc_lw     = 2
+  isol         = 2
+  elmx         = @[ELMX]
+  lwhtr        = .true.
+  swhtr        = .true.
+  cnvgwd       = @[CNVGWD]
+  cpl_imp_mrg  = @[CPL_IMP_MRG]
+  shal_cnv     = @[SHAL_CNV]
+  cal_pre      = .false.
+  redrag       = .true.
+  dspheat      = .true.
+  hybedmf      = @[HYBEDMF]
+  satmedmf     = @[SATMEDMF]
+  isatmedmf    = @[ISATMEDMF]
+  lheatstrg    = @[LHEATSTRG]
+  lseaspray    = @[LSEASPRAY]
+  random_clds  = @[RANDOM_CLDS]
+  trans_trac   = .true.
+  cnvcld       = @[CNVCLD]
+  imfshalcnv   = @[IMFSHALCNV]
+  imfdeepcnv   = @[IMFDEEPCNV]
+  do_deep      = @[DO_DEEP]
```

Comment:
The current global workflow does not have "do_deep"

Reply:
It's unlikely they'll run at a high enough resolution for do_deep=.false. to make any sense.

Reply:
We will not run high-res for the parent domain as well, right?

Reply:
@guoqing-noaa in the global_nest_v1 physics suite we will run with do_deep = true for both the high res and low res, because we use a scale-adaptive convection scheme. We set do_shal = false because we use the MYNN  scheme to do shallow convection.

Reply:
@lisa-bengtsson I did not find "do_shal".  Where to set this variable?

Reply:
Sorry my mistake, it is the shal_cnv flag above I was referring to.

Reply:
Thanks @lisa-bengtsson. To confirm, we need to set shal_cnv to false for either the global or nest domain if using the global_nest_v1 physics, right?

It looks like we set it to true in previous tests, including `/scratch2/BMC/ome/egrell/AR/v1_ufs_nested/STMP/RUNDIRS/physv1_nonest/fcst.117353/input.nml`

Reply:
@guoqing-noaa yes, since we did that testing we have changed shal_cnv from true to false in Sam's HAFS workflow. In the end, wether or not we call shallow convection in C3 or MYNN is determined by the suite definition file and not this flag, so it is a bit redundant. But I would keep it at false to be consistent with the HAFS workflow tests. (I think in the future we should aim to remove shal_cnv and do_deep since these choices are set in the SDF... but that is a different task).

Reply:
Thanks for the information, @lisa-bengtsson

Reply:
This namelist is nominally supposed to support alternative physics (not just global_nest_v1) and we certainly would have reason to turn it on for some physics suites.